gradio/interface.py

```diff
@@ -250,10 +250,10 @@ def __init__(
             self.cache_examples = False
 
         self.input_components = [
-            get_component_instance(i, render=False) for i in inputs  # type: ignore
```

Comment:
When I use the original code the repro still loads fine for me (no extra component). In which case is this change needed?

Reply:
The original behavior of setting `render=False` is the same as the current behavior of setting `unrender=True`, so yes both should have worked here.

The problem happens in `gr.Dataset` where we didn't really have a way to say -- don't render the components, but also don't unrender them. 

I also think having two explicit boolean parameters is cleaning than having a single boolean parameter that can also be `None`.